pkg/controller/controller_binding.go

```diff
@@ -370,6 +422,8 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			return err
 		}
 
+		toUpdate.Status.ExternalProperties = toUpdate.Status.InProgressProperties
```

Comment:
This method is long enough that a quality saying "Now we care handling the success case" might be helpful.